ci/tasks/build-tile.yml

```diff
@@ -1,18 +1,30 @@
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 ---
 platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: cfplatformeng/tile-generator 
+    repository: m0pt0pmatt/tile-generator
```

Comment:
Should this be changed back to the official tile generator?

Reply:
The image has an old binary, but it will be updated in the next release. https://github.com/cf-platform-eng/tile-generator/issues/290

I'll opened #223 to track this